docs/sources/flow/when-to-use-flow/opentelemetry-collector.md

```diff
@@ -0,0 +1,109 @@
+---
+description: Learn how to Flow compares to the OpenTelemetry Collector
+menuTitle: OpenTelemetry Collector
+title: OpenTelemetry Collector
+weight: 100
+---
+
+# Comparing Flow with the OpenTelemetry Collector
+
+## Which one of the two should you use? 
+
+Flow may be a better choice if:
+* You prefer River syntax over YAML.
+* You would like to collect "profile" telemetry - Flow can do this via Pyroscope components.
+* You would like to process Prometheus or Loki telemetry without converting it to OTLP.
+
+The Collector may be a better choice if:
+* You are already using the Collector, and do not need any of the features of Flow.
+* You prefer YAML syntax over River.
```

Comment:
* You have a custom component, which you can incorporate in your own Collector distribution via the Collector Builder

Reply:
Is the main point the fact that you can add a custom component? Or is using the Collector Builder the advantage?

Either way, there is nothing stopping someone from forking the Agent and adding their own custom component. We just don't document that process very well, because in the long term we want to solve by developing a plugin-based system which doesn't require a new distribution/fork when a custom component is needed.

Reply:
The use case is having a custom component, having a supported (and documented?) way to do that is what determines the tooling.